test/autorest/generated/paginggroup/pollers.go

```diff
@@ -51,7 +53,7 @@ func (p *productResultPoller) FinalResponse(ctx context.Context) (*ProductResult
 		if err != nil {
 			return nil, err
 		}
-		if res != nil && (*res.ProductResult != ProductResult{}) {
+		if res != nil {
```

Comment:
I think this block can be simplified to `return p.handleResponse(p.pt.latestResponse())`.

Reply:
Done